saleor/graphql/order/mutations/orders.py

```diff
@@ -420,33 +421,68 @@ def perform_mutation(cls, _root, info, amount, **data):
         return OrderRefund(order=order)
 
 
-class OrderUpdateMeta(UpdateMetaBaseMutation):
+class OrderUpdateMetaBaseMutation(UpdateMetaBaseMutation):
+    class Meta:
+        abstract = True
+
+    class Arguments:
+        token = graphene.UUID(
+            description="Token of an object to update.", required=True
+        )
+        input = MetaInput(
+            description="Fields required to update new or stored metadata item.",
+            required=True,
+        )
+
+    @classmethod
+    def get_instance(cls, info, **data):
+        token = data["token"]
+        return models.Order.objects.get(token=token)
+
+
+class OrderClearMetaBaseMutation(ClearMetaBaseMutation):
```

Comment:
We don't want to offer a mutation to clear the whole meta for the user. It should be allowed only for the staff user with expected permission